src/Workleap.AspNetCore.Authentication.ClientCredentialsGrant/OpenAPI/SecurityRequirementOperationFilter.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Globalization;
+using System.Text;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Workleap.AspNetCore.Authentication.ClientCredentialsGrant.OpenAPI;
+
+internal sealed class SecurityRequirementOperationFilter : IOperationFilter
+{
+    private readonly string? _audience;
+
+    public SecurityRequirementOperationFilter(IOptionsMonitor<JwtBearerOptions> jwtOptionsMonitor)
+    {
+        var jwtOptions = jwtOptionsMonitor.Get(ClientCredentialsDefaults.AuthenticationScheme);
```

Comment:
do we want to consider having `jwtOptions` as a class property similar to the DocumentFilter?